dao/src/main/resources/db/changelog/logs/ch-add-table-notification-schedule-Sannytski.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="Sannytski-1" author="Dmytro Sannytski">
+        <createTable tableName="notification_schedule">
+            <column name="notification_type" type="VARCHAR" autoIncrement="false">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="seconds" type="VARCHAR(10)">
```

Comment:
doesn't make sense to have separate columns for each character in cron expression. keep just one column.